airflow/providers/yandex/hooks/yandex.py

```diff
@@ -78,6 +78,11 @@ def get_connection_form_widgets() -> dict[str, Any]:
                 description="Optional. This key will be placed to all created Compute nodes"
                 "to let you have a root shell there",
             ),
+            "endpoint": StringField(
+                lazy_gettext("API endpoint"),
+                widget=BS3TextFieldWidget(),
+                description="Optional.",
```

Comment:
I think it's better to have here some more meaningful description

Reply:
hi! i fixed the description, if everything's ok lets merge? it seems only people with write access can do it